content/md/first-steps/manage-your-attributes.md

```diff
@@ -79,8 +79,10 @@ Each attribute has specific properties to add more control on attribute's values
 - **Reference entity single link attribute** & **reference entity multiple links attribute _(Enterprise Edition only)_**:
   - *Related reference entity*: the reference entity linked to this attribute
 - **Reference data simple select attribute** & **reference data multi select**:
-  - *Reference data type*: choose an existing reference data to link to your attribute.
-  
+  - *Reference data type*: choose an existing reference data to link to your attribute
```

Comment:
```suggestion
  - *Reference data type*: choose an existing reference entity to link to your attribute
```

Reply:
A minor point, but if "a reference data" is the right word to use instead of "entity", it needs another word at the end, such as "a reference data value"

Reply:
Yes, it's reference data, the CE feature.

Reply:
```suggestion
  - *Reference data type*: choose an existing reference data value to link to your attribute
```